package.json

```diff
@@ -42,6 +43,7 @@
     "@typescript-eslint/eslint-plugin": "^5.31.0",
     "@typescript-eslint/parser": "^5.31.0",
     "copyfiles": "^2.4.1",
+    "buffer": "^6.0.3",
```

Comment:
do we really use this dependency? if you try to remove it with `npm remove buffer` and then the app is still operating well, we can safely remove it, otherwise, let's keep it.